content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -184,18 +186,44 @@ otel:
     - b3
   resource:
     attributes:
-      environment: dev
-      xyz: foo
+      deployment.environment: dev
+      service:
+        name: cart
+        namespace: shop
 ```
 
 Environment variables example:
 
 ```shell
 export OTEL_PROPAGATORS="tracecontext,b3"
-export OTEL_RESOURCE_ATTRIBUTES="environment=dev,xyz=foo"
+export OTEL_RESOURCE_ATTRIBUTES="deployment.environment=dev,service.name=cart,service.namespace=shop"
 ```
 
-Disable the OpenTelemetry Starter:
+#### Overriding Resource Attributes
+
+As usual in Spring Boot, you can override properties in the
+`application.properties` and `application.yaml` files with environment
+variables.
+
+For example, you can change or add the `deployment.environment` resource
+attribute (not changing `service.name` or `service.namespace`) by setting the
+standard `OTEL_RESOURCE_ATTRIBUTES` environment variable:
+
+```shell
+export OTEL_RESOURCE_ATTRIBUTES="deployment.environment=prod"
+```
+
+Alternatively, you can use the Spring Boot style:
```

Comment:
I would add explanations to help the reader understand the second configuration way. I would also remove "Spring Boot style" because it may confuse some users.